src/style.css

```diff
@@ -1 +1,75 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+body {
+  padding: 0;
+}
```

Comment:
Consider using the `html` and `body` tags for setting global styles such as padding and margin. This is one of the few exceptions where styling using tag names is acceptable.